spec/controllers/track_controller_spec.rb

```diff
@@ -16,295 +16,376 @@
-    # set the time the comment event happened at to within the last week
-    ire = info_request_events(:silly_comment_event)
-    ire.created_at = Time.now - 3.days
-    ire.save!
-    user = users(:silly_name_user)
-    user.locale = "es"
-    user.save!
-    TrackMailer.alert_tracks
-    deliveries = ActionMailer::Base.deliveries
-    mail = deliveries[0]
-    expect(mail.body).to include('el equipo de ')
-  end
-end
+    it 'should destroy the track thing' do
+      get :update, {:track_id => track_thing.id,
+                    :track_medium => 'delete',
+                    :r => 'http://example.com'},
+                   {:user_id => track_thing.tracking_user.id}
+      expect(TrackThing.find(:first, :conditions => ['id = ? ', track_thing.id])).to eq(nil)
+    end
 
-describe TrackController, "when viewing RSS feed for a track" do
-  render_views
+    it 'should redirect to a URL on the site' do
+      get :update, {:track_id => track_thing.id,
+                    :track_medium => 'delete',
+                    :r => '/'},
+                   {:user_id => track_thing.tracking_user.id}
+      expect(response).to redirect_to('/')
+    end
 
-  before(:each) do
-    load_raw_emails_data
-    get_fixtures_xapian_index
+    it 'should not redirect to a url on another site' do
+      track_thing = FactoryGirl.create(:search_track)
+      get :update, {:track_id => track_thing.id,
+                    :track_medium => 'delete',
+                    :r => 'http://example.com/'},
+                   {:user_id => track_thing.tracking_user.id}
+      expect(response).to redirect_to('/')
+    end
   end
 
-  it "should get the RSS feed" do
-    track_thing = track_things(:track_fancy_dog_request)
+  describe "when sending alerts for a track" do
+    render_views
 
-    get :track_request, :feed => 'feed', :url_title => track_thing.info_request.url_title
-    expect(response).to render_template('track/atom_feed')
-    expect(response.content_type).to eq('application/atom+xml')
-    # TODO: should check it is an atom.builder type being rendered, not sure how to
+    before(:each) do
+      load_raw_emails_data
+      get_fixtures_xapian_index
+    end
 
-    expect(assigns[:xapian_object].matches_estimated).to eq(3)
-    expect(assigns[:xapian_object].results.size).to eq(3)
-    expect(assigns[:xapian_object].results[0][:model]).to eq(info_request_events(:silly_comment_event)) # created_at 2008-08-12 23:05:12.500942
-    expect(assigns[:xapian_object].results[1][:model]).to eq(info_request_events(:useless_incoming_message_event)) # created_at 2007-11-13 18:09:20.042061
-    expect(assigns[:xapian_object].results[2][:model]).to eq(info_request_events(:useless_outgoing_message_event)) # created_at 2007-10-14 10:41:12.686264
-  end
+    it "should send alerts" do
+      # set the time the comment event happened at to within the last week
+      ire = info_request_events(:silly_comment_event)
+      ire.created_at = Time.now - 3.days
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.